File: Vou.Services.GameAPI/Models/Game.cs
using System.ComponentModel.DataAnnotations;

namespace Vou.Services.GameAPI.Models
{
	public class Game
	{
		[Key]
		public int Id { get; set; }
		[Required]
		public string Name { get; set; } = string.Empty;
		[Required]
		public string Img { get; set; } = string.Empty;
		
		public bool AllowTrade { get; set; } = false;
		public string Instruction { get; set; } = string.Empty ;

	}
}

File: lib/class/game.dart
class Game {
  int id;
   String name;
   String img;
   bool allowTrade;
   String instruction;

  Game({
    required this.id,
    required this.name,
    required this.img,
    required this.allowTrade,
    required this.instruction,
  });

  factory Game.fromJson(Map<String, dynamic> json) {
    return Game(
      id: json['id'],
      name: json['name'],
      img: json['img'],
      allowTrade: json['allowTrade'],
      instruction: json['instruction'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'name': name,
      'img': img,
      'allowTrade': allowTrade,
      'instruction': instruction,
    };
  }
}
